lib/index.js

```diff
@@ -0,0 +1,84 @@
+function map(arr, fn){
```

Comment:
```suggestion
function map(arr, fn) {
```
Try to keep syntax spacing consistent.